flask_resty/view.py

```diff
@@ -308,6 +308,8 @@ def create_item(self, data):
         return self.model(**data)
 
     def add_item(self, item):
+        self.authorization.authorize_create_item(item)
```

Comment:
Should this be called `authorize_add_item` instead? We're not actually calling it in `create_item`, but instead in `add_item`. But `authorize_create_item` sounds better.

Reply:
create is more meaningful than add